scripts/install-appstudio-kcp.sh

```diff
@@ -0,0 +1,308 @@
+#!/bin/bash
+#
+# exit immediately when a command fails
+set -e
+# only exit with zero if all commands of the pipeline exit successfully
+set -o pipefail
+# error on unset variables
+set -u
+
+command -v kubectl >/dev/null 2>&1 || { echo "kubectl is not installed. Aborting."; exit 1; }
+command -v oc >/dev/null 2>&1 || { echo "oc cli is not installed. Aborting."; exit 1; }
+
+# tr is not working by default in MacOs. Exporting LC_CTYPE=C seems like solve the problem
+case "$(uname -s)" in
+    Darwin*)    export LC_CTYPE=C;;
+    Linux*)     echo -e "Running on Linux system";;
+    *)          echo -e "UNKNOWN System."
+esac
+
+export ROOT_E2E="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..
+export WORKSPACE=${WORKSPACE:-${ROOT_E2E}}
+export MY_GIT_FORK_REMOTE="qe"
+export MY_GITHUB_ORG="redhat-appstudio-qe"
+export MY_GITHUB_TOKEN="$GITHUB_TOKEN"
+export WORKSPACE_ID=$(tr -dc a-z0-9 </dev/urandom | head -c 5 ; echo '')
+export TEST_BRANCH_ID="$(date +%s)"
+export JOB_TYPE=${JOB_TYPE:-"presubmit"}
+export REPO_NAME=${REPO_NAME:-"e2e-tests"}
+export RUN_E2E="false"
+
+# KCP related environments
+export KCP_CONTEXT=
+export KCP_KUBECONFIG=
+export CLUSTER_KUBECONFIG=
+export ROOT_WORKSPACE=
+export IS_STABLE="false"
+export APPSTUDIO_WORKSPACE="redhat-appstudio-${WORKSPACE_ID}"
+export HACBS_WORKSPACE="redhat-hacbs-${WORKSPACE_ID}"
+export USER_APPSTUDIO_WORKSPACE="appstudio-${WORKSPACE_ID}"
+export COMPUTE_WORKSPACE="compute-${WORKSPACE_ID}"
+export PIPELINE_SERVICE_SP_WORKSPACE="root:redhat-pipeline-service-compute"
+export PIPELINE_SERVICE_IDENTITY_HASH="72b2990e51b1931e9fee86e67091b721a8c32f407d762fc847d9d2316a988b52"
```

Comment:
in the description it is mentioned to export these vars, however they are hardcoded here...

Reply:
My bad.